packages/@o3r/rules-engine/builders/rules-engine-extractor/helpers/rules-engine.extractor.ts

```diff
@@ -316,7 +316,7 @@ export class RulesEngineExtractor {
         });
       }
     });
-    return operators;
+    return Array.from((new Map(operators.map((operator) => [operator.id, operator]))).values());
```

Comment:
at this level I wonder if a simple filter would not be faster and more readable:
```typescript
return operators
  .fitler(({id}, index, arr) => !arr.slice(index).some((o) => o.id === id));
```
but I am fine with it :)